<launch>

<node pkg="offboard_control" name="Quad1_offboard_pvacontrol" type="pva_uav1_Target.py" output="screen" >
</node>

<node pkg="offboard_control" name="Quad2_offboard_pvacontrol" type="pva_uav2_Seeker1.py" output="screen" >
</node>

<node pkg="offboard_control" name="Quad3_offboard_pvacontrol" type="pva_uav3_Seeker2.py" output="screen" >
</node>

<node pkg="offboard_control" name="Quad4_offboard_pvacontrol" type="pva_uav4_Seeker3.py" output="screen" >
</node>


</launch>